soc/microchip/mec/common/reg/mec_adc.h

```diff
@@ -10,9 +10,16 @@
 #include <stdint.h>
 #include <stddef.h>
 
+#if defined(CONFIG_SOC_MEC172X_NLJ) || defined(CONFIG_SOC_MEC1743_QLJ) ||		\
```

Comment:
NO. Please DO NOT add MEC1743 and MEC1753 to this code!
MEC174x, MEC175x, and all future chips will be based on a the new MEC5 HAL and will NOT EVER use the old MEC172x drivers.

Reply:
The new MEC5 HAL is a work in progress.
The HAL was created due to the request of yet another large customer.  The original goal for MEC5 HAL were all chips after MEC172x.  Since the customer demanded no direct HW register access in the drivers we can't use the current MEC172x drivers.  This meant new SoC, boards, drivers, etc as if MEC5 are completely different chips.  Unfortunately, said customer came back and demanded MEC172x be added to MEC5 HAL.  It is a huge mess with no good solution so the rule is:
MEC152x uses old header files from hal_microchip/mec folder
MEC172x does not use any header/code from hal_microchip due to Google chromebook Zephyr requirement.
MEC174x, MEC175x, and hopefully all future chips will use hal_microchip/mec5 headers and peripheral C code.
MEC174x, MEC175x require new SoC, DTS, board, and drivers.
Their are only a new select header files that may be shared:  the GPIO dt-bindings and PINCTRL dt-bindings.  Anything else is not shared because we don't want to break legacy MEC172x.
No one is happy about it.